src/content/guides/getting-started.md

```diff
@@ -69,24 +69,23 @@ __index.html__
 </html>
 ```
 
-In this example, there are implicit dependencies between the `<script>` tags. Our `index.js` file depends on `lodash` being included in the page before it runs. This is because `index.js` never declared a need for `lodash`; it just assumes that the global variable `_` exists.
+In this example, there are implicit dependencies between the `<script>` tags. The `index.js` file depends on including `lodash` in the page before it runs. This is because `index.js` never declared a need for `lodash`; it assumes that the global variable `_` exists.
 
 There are problems with managing JavaScript projects this way:
 
-- It is not immediately apparent that the script depends on an external library.
-- If a dependency is missing, or included in the wrong order, the application will not function properly.
-- If a dependency is included but not used, the browser will be forced to download unnecessary code.
+-   It is not immediately apparent that the script depends on an external library.
```

Comment:
No need for double spaces here?

Reply:
Again it's my markdown linter, apparently it's 'more correct', but somewhat irrelevant.

Reply:
This can bring more fragmentation to the content, so I would suggest reversing these type of changes since they don't add value and make the PR even bigger and longer to review :)